app/views/repositories/index.html.erb

```diff
@@ -21,6 +21,7 @@
         <i class="fab fa-github"></i>
         <span class="d-inline-block"><%= repository.link %></span>
         <span class="d-inline-block float-right"> <%= repository.languages %></span>
+        <span class="d-block text-dark"><%= simple_format repository.description %></span>
```

Comment:
I am trying to remember this method; where is it coming from?

Reply:
It's a helper from ActionView:
https://api.rubyonrails.org/classes/ActionView/Helpers/TextHelper.html